sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceSubscriptionKey.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// A <see cref="ServiceSubscriptionKey"/> is a subscription key use to authenticate
+    /// against a Cognitive Service.
+    /// </summary>
+    public class ServiceSubscriptionKey
```

Comment:
Per the updated guidelines, let's call this TextAnalyticsSubscriptionKeyCredential.  I don't see it referred to as API key anywhere in the docs, and it is heavily referred to as subscription key.